### ISSUE TITLE
Example needs updating

### ISSUE THREAD

**Issue by arj03** [dominictarr/ssb-thread#2]
Seems one is required to add an unbox function now?

**Comment by dominictarr** [comment]
hmm, and looking at the code it also assumes you'll be able to decrypt the thread...
which probably isn't always reasonable.

**Comment by arj03** [comment]
Yeah, furthermore, it seems that the return is array and not err, array.
